### PR TITLE
Changed all instances of 'money' found in the docs

### PR DESCRIPTION
**Pull request by angusjoshi**
Tried to change to an appropriate alternative based on context.
Addressing #13338

### CODE REVIEW

File: docs/examples/safe-remote.rst [hunk 1]
```diff
@@ -6,18 +6,18 @@ Safe Remote Purchase
 
 Purchasing goods remotely currently requires multiple parties that need to trust each other.
 The simplest configuration involves a seller and a buyer. The buyer would like to receive
-an item from the seller and the seller would like to get money (or an equivalent)
+an item from the seller and the seller would like to get currency
```
Comment (ekpyron) [inline docs/examples/safe-remote.rst]: Hm... I'm not sure "currency" is that much better than "money"... maybe we should go for "some compensation, e.g. Ether," in the first occurrence here and then move to "Ether" in the further occurrences below (those refer to the example in particular, in which it *is* Ether)...

**Review by ekpyron — COMMENTED** [review]
Thank you very much! This looks good, except for the one case I commented about below.

File: docs/examples/safe-remote.rst (hunk 1 above)
Comment (angusjoshi) [reply docs/examples/safe-remote.rst]: Makes sense, just committed this change.

Comment (ekpyron) [reply docs/examples/safe-remote.rst]: Can you also change the other occurrences of "currency"? (if I see it correctly "Ether" should fit for those)

Comment (angusjoshi1) [reply docs/examples/safe-remote.rst]: Shall I change all instances of currency outwith the changes I made as well?

Comment (angusjoshi) [reply docs/examples/safe-remote.rst]: done.

**Comment by cameel** [comment]
Needs rebase on latest `develop` to have the OZ test pass. And the chainlink one looks just like a transient networking error.

**Comment by angusjoshi** [comment]
done this rebase.

**Comment by ekpyron** [comment]
This looks good now, but from the looks of it you *merged* develop instead of rebasing it :-).
If you can rebase it with ``git rebase -i develop`` and simultaneously "squash" the changes into a single commit, then we can merge (otherwise we can also take care of the rebase).

**Comment by ekpyron** [comment]
Damn it, something went wrong here now...

**Comment by ekpyron** [comment]
I tried to rebase this against the fork (which usually works), but apparently I lost the changes while doing so, I'm very sorry about that...

**Comment by ekpyron** [comment]
@angusjoshi1 Do you happen to still have the changes locally? If so you can just push them again using ``git push -f`` and we can take it from there. Again sorry about my git accident here :-).

**Pull request merged**